lib/src/builder/generation/impl_generation.dart

```diff
@@ -727,9 +727,12 @@ class ImplGenerator {
         ..writeln('  @override')
         ..writeln('  bool get \$isClassGenerated => true;')
         ..writeln()
+        ..writeln('  var _factoryOverride;')
```

Comment:
This should be typed. 

I know we won't hit runtime type errors since we're only using this in generated code, but it will be good to have that type information in case it helps optimize the dart2js output, since this code path gets hit a lot.